portfolio.html

```diff
@@ -17,21 +17,20 @@
 
 <!--------Navbar-------->
 
-    <!--------Video-------->
 
     <div class="video_container">
         <video src="video/99544-1080.mp4" type="video/mp4" loop autoplay></video>
     </div>
 
     <nav class="nav_container">
         <div class="personal_logo">
-            <img class="personal_logo" src="images/logo_transparent.png" alt="personal logo">
+            <img class="logo_img" src="images/logo_transparent.png" alt="personal logo">
         </div>
 
         <div class="nav_bar">
-            <a href="#about_me" title="Link to about me section">About Me</a>
-            <a href="#portfolio" title="Link to portfolio section">Portfolio</a>
-            <a href="#contact" title="Link to contact section">Contact</a>
+            <a class="nav_link" href="#about_me" title="Link to about me section">About Me</a>
+            <a class="nav_link" href="#portfolio" title="Link to portfolio section">Portfolio</a>
```

Comment:
the portfolio link didn't seem to work for me

Reply:
Yes I haven't connected it yet. Will do so now.